corehq/apps/events/migrations/0001_initial.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.16 on 2023-02-23 08:46
```

Comment:
Nit: Should this migration be called `initial` rather than something like `add_event_table`

Reply:
Yeah, that might be more helpful. Will make update shortly